Microsoft.Toolkit.Win32.UI.Controls/Interop/Win32/UnsafeNativeMethods.cs

```diff
@@ -0,0 +1,87 @@
+// ******************************************************************
+// Copyright (c) Microsoft. All rights reserved.
+// This code is licensed under the MIT License (MIT).
+// THE CODE IS PROVIDED “AS IS”, WITHOUT WARRANTY OF ANY KIND, EXPRESS OR IMPLIED,
+// INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+// FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT.
+// IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM,
+// DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT,
+// TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH
+// THE CODE OR THE USE OR OTHER DEALINGS IN THE CODE.
+// ******************************************************************
+
+using System;
+using System.ComponentModel;
+using System.Runtime.InteropServices;
+using System.Security;
+using System.Text;
+
+namespace Microsoft.Toolkit.Win32.UI.Controls.Interop.Win32
+{
+    internal static class UnsafeNativeMethods
+    {
+        // Critical: P-Invokes
```

Comment:
@normesta @mcleanbyron I tried as much as possible to annotate these as I went along, but there are quite a few that I need to document and many, many more attributes I need to add to make this a level 2 library. Right now we only allow full trust because 1) the way we _don't_ check permissions on apps using the bridge is odd, and 2) the web view host has über permissions.